### PR TITLE
Add Learning WG

### PR DESCRIPTION
**Pull request by spastorino**
@nikomatsakis @davidtwco I've opened this PR as a starting point so we can discuss.

### CODE REVIEW

File: working-groups/learning/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- Ensure that API doc coverage is at least 90%
+
+# How can I get involved?
+If you are interested in getting involved in this working group, you should try attend a meeting and
+introduce yourself or send a message in the Zulip stream. You can be added to the GitHub and Zulip
+group for the working group if you are interested in being pinged when there are available tasks.
+
+## Meetings
+We follow a "sprint-like" structure where we on each "sprint" cycle we start with a planning and
+then work towards the goals defined on that plan.
```
Comment (spastorino) [inline working-groups/learning/README.md]: At some point we need to define a meeting time.

File: README.md [hunk 2]
```diff
@@ -88,6 +89,7 @@ Name                                                      | Status       | Short
 [mir-opt-stream]: https://rust-lang.zulipchat.com/#narrow/stream/189540-t-compiler.2Fwg-mir-opt
 [pipelining-stream]: https://rust-lang.zulipchat.com/#narrow/stream/195180-t-compiler.2Fwg-pipelining
 [polonius_stream]: https://rust-lang.zulipchat.com/#narrow/stream/186049-t-compiler.2Fwg-polonius
+[learning_stream]: https://rust-lang.zulipchat.com/#narrow/stream/<wg-learning-stream>
```
Comment (spastorino) [inline README.md]: We need to open a stream, I don't have rights to do so.

File: working-groups/learning/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- **Desired experience level:** Any
+- **Relevant repositories:** [`rust-lang/rust`][rust-repo] and [rust-lang/rustc-guide][rustc-guide-repo]
+- **Zulip stream:** [`#t-compiler/wg-learning`][zulip] on Zulip
+
+[rust-repo]: https://github.com/rust-lang/rust
+[rustc-guide-repo]: https://github.com/rust-lang/rustc-guide
+[zulip]: https://rust-lang.zulipchat.com/#narrow/stream/<wg-learning-stream>
```
Comment (spastorino) [inline working-groups/learning/README.md]: Same, we need to open a stream, I don't have rights to do so.

File: working-groups/learning/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Learning Working Group
+![working group status: active][status]
+
+This working group aims to make compiler easier to learn by ensuring that rustc-guide and api docs are "complete".
```
Comment (igaray) [inline working-groups/learning/README.md]: the compiler*

File: working-groups/learning/NOTES.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Learning Meeting Notes
+
+This document will contain meeting notes from the diagnostics working group.
```
Comment (davidtwco) [inline working-groups/learning/NOTES.md]:
```suggestion
This document will contain meeting notes from the learning working group.
```

File: README.md (hunk 2 above)
Comment (davidtwco) [reply README.md]: https://rust-lang.zulipchat.com/#narrow/stream/196385-t-compiler.2Fwg-learning

File: working-groups/learning/README.md (hunk 3 above)
Comment (davidtwco) [reply working-groups/learning/README.md]: https://rust-lang.zulipchat.com/#narrow/stream/196385-t-compiler.2Fwg-learning

File: README.md (hunk 2 above)
Comment (nikomatsakis) [reply README.md]:
```suggestion
[learning_stream]: https://rust-lang.zulipchat.com/#narrow/stream/196385-t-compiler.2Fwg-learning
```

Comment (spastorino) [reply README.md]: Done

File: working-groups/learning/README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+[ideas]: https://paper.dropbox.com/doc/Learning-Working-Group-Ideas--Ab18Mvvg7fYl8b~WL_klryqnAg-2R49k1GWZFojHIQ5x1fuB
+
+## What is the goal of this working group?
+This working group aims to accomplish the following:
+
+- Ensure that major components of rustc are covered in rustc-guide
+- Ensure that API doc coverage is at least 90%
+
+# How can I get involved?
+If you are interested in getting involved in this working group, you should try attend a meeting and
```
Comment (nikomatsakis) [inline working-groups/learning/README.md]: We don't have any meetings, do we?

Comment (nikomatsakis) [reply working-groups/learning/README.md]: Let's just say "come and say hi in the zulip stream" then

**Pull request merged**